### PR TITLE
Changed view of menu.

### PR DESCRIPTION
**Pull request by MarynaSirenko**
Branch is ready to review. @Vyhyrynskyi

### CODE REVIEW

File: Menu.py [hunk 1]
```diff
@@ -25,9 +25,10 @@ def get_total_mark(info):
     print(name,'total',total_mark)
 
 information = ' '
```
Comment (Vyhyrynskyi) [inline Menu.py]: We don't need this variable anymore. Please, delete this line.

Comment (Vyhyrynskyi) [inline Menu.py]: We don't need this variable anymore. Please, delete this line.

**Review by Vyhyrynskyi — COMMENTED** [review]
Test comment.

File: Menu.py (hunk 1 above)
Comment (MarynaSirenko) [reply Menu.py]: Done!

**Pull request merged**